utils.mk

```diff
@@ -160,3 +180,62 @@ standard_rust_check:
 	cargo clippy --all-targets --all-features --release \
 		-- \
 		-D warnings
+
+# Install a file
+#
+# params:
+# $1 : file to install
+# $2 : directory path where file will be installed
+# $3 : permissions to apply to the installed file.
+define INSTALL_FILE
+    install \
+        --mode $3 \
+        --owner $(KATA_INSTALL_OWNER) \
+        --group $(KATA_INSTALL_GROUP) \
+        -D $1 $2/$(notdir $1) || exit 1;
+endef
+
+# Install an executable file.
+#
+# params:
+# $1 : file to install
+# $2 : directory path where file will be installed
+define INSTALL_EXEC
```

Comment:
`INSTALL_EXEC` and `INSTALL_CONFIG` used to honor `DESTDIR` so that the caller didn't need to explicitly prefix in the target path parameter. I'm wondering what's the rationale behind that change of behavior.

Implicitly prefixing `DESTDIR` makes it less flexible, however more robust as in the future a new caller might forget to consider `DESTDIR` .